iron-overlay-behavior.html

```diff
@@ -265,6 +280,14 @@
     },
 
     _withBackdropChanged: function() {
+      // If tabindex is already set, no need to override it.
+      if (this.withBackdrop && !this.hasAttribute('tabindex')) {
+        this.setAttribute('tabindex', '-1');
+        this.__shouldRemoveTabIndex = true;
+      } else if (this.__shouldRemoveTabIndex) {
+        this.removeAttribute('tabindex');
+        this.__shouldRemoveTabIndex = false;
+      }
```

Comment:
Would it be sufficient to record the previous value of `tabindex` (if any) and then set it back if it is `!= null` when `withBackdrop` is `false`?